README.md

```diff
@@ -94,6 +94,26 @@ Kubernetes Operator. Learn more in the
 
 ---
 
+## Developer notes: Using a custom thv binary (dev only)
+
+During development, you can test the UI with a custom `thv` binary by running it
+manually:
+
+1. Start your custom `thv` binary with the serve command:
+
+   ```bash
+   thv serve --openapi --host=127.0.0.1 --port=50000
+   ```
+
+2. Set the `THV_PORT` environment variable and start the dev server:
+   ```bash
+   THV_PORT=50000 pnpm start
+   ```
+
+The UI will display a warning banner showing the HTTP address it's connecting to
+when using a custom port. This only works in development mode; packaged builds
+always use the embedded binary.
+
```

Comment:
I would probably leverage only the /doc/README file that is specilized on developer guide